hawkular-component/src/main/java/org/hawkular/metrics/security/InfluxAuthHttpHandler.java

```diff
@@ -41,6 +40,12 @@
  * @author Thomas Segismont
  */
 public class InfluxAuthHttpHandler implements HttpHandler {
+    private static final Pattern UUID_PATTERN;
+
+    static {
```

Comment:
No reason except that it keeps things in one line. No too much important 
to me.

Le 05/02/2016 11:23, Juraci Paixão Kröhling a écrit :
> In
> hawkular-component/src/main/java/org/hawkular/metrics/security/InfluxAuthHttpHandler.java
> <https://github.com/hawkular/hawkular-metrics/pull/444#discussion_r51998487>:
>
>> @@ -41,6 +40,12 @@
>>   * @author Thomas Segismont
>>   */
>>  public class InfluxAuthHttpHandler implements HttpHandler {
>> +    private static final Pattern UUID_PATTERN;
>> +
>> +    static {
>
> Just out of curiosity: why is this wrapped into a static block? You have
> only one statement and there's no exception to catch, so, it has the
> same effect as if you were assigning directly at the declaration above, no ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hawkular/hawkular-metrics/pull/444/files#r51998487>.
>